modules/post/bsd/gather/hashdump.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'metasploit/framework/hashes/identify'
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Linux::Priv
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super( update_info( info,
+      'Name'          => 'BSD Dump Password Hashes',
+      'Description'   => %q{ Post module to dump the password hashes for all users on a BSD system. },
+      'License'       => MSF_LICENSE,
+      'Author'        => ['bcoles'],
+      'Platform'      => ['linux'],
```

Comment:
`bsd`?

Reply:
If sessions are misidentified, you could add both. I see you're pulling in `Msf::Post::Linux::Priv`.

Reply:
Pretty sure this was missed during copypasta and wasn't causing conflicts

Reply:
Fixed and tested the `platform`.

The Linux libs `Msf::Post::Linux::Priv` are used because we don't have BSD equivalents.